packages/ui/src/elements/Table/DefaultCell/fields/File/index.tsx

```diff
@@ -26,7 +27,8 @@ export const FileCell: React.FC<FileCellProps> = ({
           ...rowData,
           filename,
         }}
-        fileSrc={rowData?.thumbnailURL || rowData?.url}
+        fileSrc={src}
+        key={src}
```

Comment:
Due to a side effect of the `hasPreloaded` state in `Thumbnail`

before: 

https://github.com/user-attachments/assets/5e13579e-c9fd-4f7e-a4a4-b29ba0987201

after:

https://github.com/user-attachments/assets/c637d97c-cf24-47a4-89b4-21d10b14135f